apps/server/src/modules/tool/context-external-tool/service/context-external-tool.service.ts

```diff
@@ -76,4 +77,39 @@ export class ContextExternalToolService {
 			throw new RestrictedContextMismatchLoggable(externalTool.name, contextExternalTool.contextRef.type);
 		}
 	}
+
+	public async copyContextExternalTool(
```

Comment:
I dont know how much tools we would have in a course or on a board, but we should test a copying a bunch of it